packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -943,6 +950,19 @@ export const Table = <
                           (c) => c.id !== SELECTION_CELL_ID, // first non-selection column is the expander column
                         );
 
+                    // override "undefined" min-width with default value
+                    if (column.minWidth === undefined) {
+                      column.minWidth = COLUMN_MIN_WIDTHS.default;
+                    }
+
+                    // expander column should be wider to accommodate the expander icon
+                    if (
+                      columnHasExpanders &&
+                      column.minWidth === COLUMN_MIN_WIDTHS.default
+                    ) {
+                      column.minWidth = COLUMN_MIN_WIDTHS.withExpander;
+                    }
```

Comment:
Can/Should these two if statements be combined?

Reply:
Why/how? They are doing completely different things based on different conditions and should be run sequentially

Reply:
Why: Testing `column.minWidth === COLUMN_MIN_WIDTHS.default` in the second if statement felt unnatural to me because if the `column.minWidth` is set to `72` by the user, I guess we should not override their value.

How: I was thinking of something like this:

```tsx
if (column.minWidth === undefined) {
  if (columnHasExpanders) {
    column.minWidth = COLUMN_MIN_WIDTHS.withExpander;
  } else {
    column.minWidth = COLUMN_MIN_WIDTHS.default;
  }
}
```

Reply:
That's not the same thing. I am explicitly increasing the value of `minWidth` from 72 (set at the top of this component) to 108 for the expander column. But I see what you're saying. Change made in [`f4a99d1`](https://github.com/iTwin/iTwinUI/pull/1863/commits/f4a99d1ec880fe899d7a9e9722c23973dd790a23)

Reply:
> 72 (set at the top of this component)

Not sure I understand where you are setting it to `72`/`COLUMN_MIN_WIDTHS.default`.

But yes, the updated code is cleaner.

Reply:
![image](https://github.com/iTwin/iTwinUI/assets/9084735/36f2d621-d751-4ed5-bfc0-fdacbd57fbca)

Reply:
So, if you initialized it at the top, was the first if statement (`if (column.minWidth === undefined) {`) not needed?

Reply:
The user could pass `undefined` explicitly, in which case we also want to override it.